packages/controllers/src/hooks.rs

```diff
@@ -121,6 +121,19 @@ impl<'a> Hooks<'a> {
         let hooks = hooks.into_iter().map(String::from).collect();
         Ok(HooksResponse { hooks })
     }
+
+    // Returns true is this_hook is in hooks; false otherwiseq
```

Comment:
```suggestion
    // Returns true is this_hook is in hooks
```
Never mind the typo, but there's no need in stating the obvious.